buildSrc/src/main/resources/minecraft_specific_words.txt

```diff
@@ -6,6 +6,7 @@ gameplay
 griefing
 minceraft
 minecraft
+datapack
```

Comment:
Wasn't this removed in favor of always using "data pack"?

Reply:
you're right, I'll fix it